docs/conf.py

```diff
@@ -202,3 +201,5 @@
 myst_enable_extensions = [
     "colon_fence",
 ]
+
+myst_all_links_external = True
```

Comment:
This ensures the myst_parser parses all links as hyperlinks. This will remove all the warnings related to links pointing to external files.